src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -497,4 +497,7 @@
         Items="@(CorrelationFile)"
         Overwrite="true" />
   </Target>
+
+  <!-- Setup Dumpling service to collect and upload any crash dumps -->
+  <Import Project="$(MSBuildThisFileDirectory)Dumpling.targets" />
```

Comment:
@MattGal @mellinoe  I imported this target in two places:
1) `tests.targets` (only with property EnableDumpling=true), with the goal of enabling this for local non-Helix test runs
2) `CloudTest.targets` (always), with the goal of enabling this on all Helix test runs

I'm not sure if I'm actually importing this in the right place to achieve my goals, though.